the autotrace library uses these libraries, not the autotrace binary

--- Makefile.am
+++ Makefile.am
@@ -105,6 +105,12 @@
 		exception.h
 
 libautotrace_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+libautotrace_la_LIBADD = \
+		$(MAGICK_LDFLAGS)		\
+		$(LIBPNG_LDFLAGS)		\
+		$(LIBSWF_LDFLAGS)		\
+		$(LIBPSTOEDIT_LIBS)		\
+		-lm
 
 #
 # noinst_HEADERS: headers shared between lib and bin.
@@ -126,13 +132,7 @@
 		strgicmp.h
 
 INCLUDES = $(MAGICK_CFLAGS) $(LIBPSTOEDIT_CFLAGS)
-autotrace_LDADD =				\
-		libautotrace.la			\
-		$(MAGICK_LDFLAGS)		\
-		$(LIBPNG_LDFLAGS)		\
-		$(LIBSWF_LDFLAGS)		\
-		$(LIBPSTOEDIT_LIBS)		\
-		-lm
+autotrace_LDADD = libautotrace.la
 
 m4datadir = $(datadir)/aclocal
 m4data_DATA = autotrace.m4
